### ISSUE TITLE
Question: what is the syntax to point to a MARC indicator?

### ISSUE THREAD

**Issue by phochste** [MARCspec/MARC-Spec#2]
To point to a 245 field with indicator1 == 1 one could use the MARC spec:

```
245_1
```

To point to a 245$a subfield with indicator1 == 1 one could use the MARC spec:

```
245_1$a
```

Question: is it possible to very explicitly point to the value of the first indicator in 245 (not to the field value or the subfield value)? Or, is this dependent on the context of use when working with MARC spec. E.g. in hypothetical functions:

```
 remove_field(245_1)            <- 245_1 is the MARC 245 field with indicator1 == 1
 remove_indicator(245_1)     <- 245_1 is the MARC 245 indicator1. (not the field).
```

**Comment by cKlee** [comment]
With MARCspec in Catmandu you can not point directly to the value of an indicator. Indicators are always context to the field value. In Catmandu I made a compromise: Instead of referring to the whole field data, something like '245' refers to the data content (values) of the field rather than to the whole data (tag, indicators, positions, subfield codes and values). So this information (indicators) get lost and they are reduced to filter functionality. 
I thought about returning an object with all the data, but that doesn't seem to fit in the Catmandu logic. But it's not to late to rethink this.

**Comment by cKlee** [comment]
solved with version 2.0: 

    remove_indicator(245^1{245^1=\1})
